klayout/drc/rule_decks/lres.drc

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
+  #================================================
+  #----------------N+ POLY RESISTOR----------------
+  #================================================
+
+  logger.info('Starting LRES derivations')
+  lres_poly = poly2.and(nplus).interacting(sab).interacting(res_mk)
```

Comment:
should we save `poly2.and(nplus)` to an intermediate variable? it seems that it is repeated below.

Reply:
Done

Reply:
Thanks @FaragElsayed2